### DISCUSSION TITLE
what version do i have to install?

### DISCUSSION THREAD

**Discussion by pawelpietryka** [Martchus/syncthingtray#251] [General]
I'm on windows 10 with intel CPU. Thanks!

While this might be obvious to developers I think this list below is overkill/confusing 

syncthingctl-1.5.2-i686-w64-mingw32.exe.zip
syncthingctl-1.5.2-i686-w64-mingw32.exe.zip.sig
syncthingctl-1.5.2-x86_64-pc-linux-gnu.tar.xz
syncthingctl-1.5.2-x86_64-pc-linux-gnu.tar.xzsig
syncthingctl-1.5.2-x86_64-w64-mingw32.exe.zip
syncthingctl-qt5-1.5.2-i686-w64-mingw32.exe.zip
syncthingctl-qt5-1.5.2-i686-w64-mingw32.exe.zipsig
syncthingctl-qt5-1.5.2-x86_64-w64-mingw32.exe.zip
syncthingtray-1.5.2-i686-w64-mingw32.exe.zip
syncthingtray-1.5.2-i686-w64-mingw32.exe.zip.sig
syncthingtray-1.5.2-x86_64-pc-linux-gnu.tar.xz
syncthingtray-1.5.2-x86_64-w64-mingw32.exe.zip
syncthingtray-qt5-1.5.2-i686-w64-mingw32.exe.zip
syncthingtray-qt5-1.5.2-i686-w64-mingw32.exe.zi psig
syncthingtray-qt5-1.5.2-x86_64-w64-mingw32.exe.zip
syncthingtray-qt5-1.5.2-x86_64-w64-mingw32.exe.zip.sig

**Comment by Martchus** [comment]
Most likely syncthingtray-1.5.2-x86_64-w64-mingw32.exe.zip but you didn't provide enough information to tell.

I know this list is subnormal but creating a website with common downloads that perhaps even takes the user agent into account hasn't been done yet.

**Comment by Martchus** [comment]
I was drafting a web page for Syncthing recently. It is not done yet but the downloads section will hopefully solve this problem: https://martchus.github.io/syncthingtray/#downloads-section

So far it only expands the relevant section for your user agent but it would also make sense to nail it down to the one concrete suggestion that also takes your CPU architecture into account.

(This page is still really WIP, e.g. it lacks a proper loading indication and besides the downloads page there's not much content to be seen.)

**Comment by Martchus** [comment]
You may want to try the new website to download the new version I've just released: https://martchus.github.io/syncthingtray/#downloads-section

I hope it is less confusing while still conveying enough technical details to make the best decision depending on one's system.

**Comment by Nordlicht-13** [comment]
You can install Syncthing Tray using winget:
https://www.youtube.com/watch?v=JjD4VXEjr_U
Then you can autoupdate it via WAO WingetAutoUpdater:
https://www.youtube.com/watch?v=AU3OgVIWPAw

(The mentioned YouTube videos are German.)

**Reply by Martchus** [reply]
Yes, the winget package is mentioned on the website (as well as caveats regarding community-provided packaging in general).
